### PR TITLE
feat(concepts): suspend free trial

### PR DESCRIPTION
**Pull request by Sijoma**
## Description

The disks are getting deleted on a trial suspension. This needs to be explained in order to prevent confusion and explain what are actions the user then needs to perform to get their processes running again. 
This also explains what is actually retained for the cluster. 

@mesellings although this is a minor change, I would be still interested for your input. The concept of "suspension but still deleting disks" could be a bit confusing & surprsing.

This is also version independent. Do we back port concepts? 

Documentation for PDP: https://github.com/camunda/product-hub/issues/2548
https://github.com/camunda-cloud/camunda-operator/issues/3412

## When should this change go live?

We target a release mid-next week. I would not object to already release it Monday before we rollout the change to decouple the timeline a bit. 

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [x] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by mesellings** [comment]
@Sijoma I've committed some edits - if you are happy with these, I can backport the changes, approve the PR, and get this published 👍

**Review by mesellings — DISMISSED** [review]
Lgtm 👍 🚀

**Comment by Sijoma** [comment]
> @Sijoma I've committed some edits - if you are happy with these, I can backport the changes, approve the PR, and get this published 👍

looks good! TY

**Comment by mesellings** [comment]
To be merged and docs released Tuesday 19/Weds 20 depending on QA.

File: versioned_docs/version-8.2/components/concepts/clusters.md
Comment (akeller) [inline versioned_docs/version-8.2/components/concepts/clusters.md:1]: 8.2 is technically no longer supported, so this change wouldn't apply here unless it was available before 8 Oct, 2024.

Comment (mesellings) [reply versioned_docs/version-8.2/components/concepts/clusters.md:1]: Thank you, good spot! I've removed the 8.2 changes from this PR 👍

**Review by mesellings — APPROVED** [review]
LGTM 👍 🚀

**Pull request merged**